src/index.css

```diff
@@ -1,27 +1,50 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  font-family: "Lato", sans-serif;
+  transition: 0.8s;
+  margin: 0 0;
+}
+
 body {
-  margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  background-color: #dedbd8;
+}
+
+#root {
+  width: 30%;
+  margin: auto;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+}
+
```

Comment:
- [ ] It would be better to use the `width` property to fix it. (I ran the app in different browsers so that is not the problem.).  (If you see that your project works well on your site maybe there are some changes that you forgot to submit. However, it's just a guess)

- The video shows an option to fix it. ♻️ 

https://user-images.githubusercontent.com/79876959/144184308-f296d3bd-a235-4e9c-9861-8b49593bcd79.mp4

Reply:
Hi @ArthurGC ... it's been corrected!

Reply:
This issue is well resolved! ✔️